### PR TITLE
KIALI-GH-1211 Improve layout popups

### PR DESCRIPTION
**Pull request by jshaughn**
- Also, add tooltips for zoom options

This is for one of the bullets in https://github.com/kiali/kiali/issues/1211.

kiali-gh-1211

### CODE REVIEW

**Comment by jshaughn** [comment]
cc @abonas

File: src/components/CytoscapeGraph/CytoscapeToolbar.tsx [hunk 1]
```diff
@@ -64,16 +64,16 @@ export class CytoscapeToolbar extends React.PureComponent<CytoscapeToolbarProps>
     return (
       <div className={cytoscapeToolbarStyle}>
         <ButtonGroup>
-          <Button onClick={this.zoomIn}>
+          <Button onClick={this.zoomIn} title="zoom in">
```
Comment (josejulio) [inline src/components/CytoscapeGraph/CytoscapeToolbar.tsx]: I think we should capitalize the first word, e.g. `Zoom in` and the same for the rest of the titles.

Comment (mtho11) [inline src/components/CytoscapeGraph/CytoscapeToolbar.tsx]: I think we want to use the full-on PF Tooltips. They allow us nicer formatting and font choices (and even icons if we want)

Comment (jshaughn) [reply src/components/CytoscapeGraph/CytoscapeToolbar.tsx]: ok, I mean OK

Comment (jshaughn) [reply src/components/CytoscapeGraph/CytoscapeToolbar.tsx]: I just knew someone would ask for that...

**Comment by jshaughn** [comment]
@mtho11 @josejulio The video and PR have been updated. Are you OK with the tooltip text?

**Pull request merged**